devito/equation.py

```diff
@@ -24,6 +28,9 @@ class Eq(sympy.Eq):
     subdomain : SubDomain, optional
         To restrict the computation of the Eq to a particular sub-region in the
         computational domain.
+    coefficients : Coefficients, optional
```

Comment:
reminder, not necessarily for this PR, but quite important: we need to add examples to the docstring, showing how to use SubDomain as well as Coefficient

Reply:
I can add an example (within the next couple of days) in a different PR?